.github/ISSUE_TEMPLATE/bugReportForm.yml

```diff
@@ -0,0 +1,28 @@
+name: Bug Report
+description: File a bug report, we will respond to this thread with any questions.
+title: 'Bug: '
+labels: ['bug', 'triage']
+assignees: '@Azure/aks-atlanta'
+body:
+   - type: input
+     id: What-happened
+     attributes:
+        label: What happened?
+        description: Tell us what did you expect to happen?
```

Comment:
super nit: would you mind changing this to "Tell us what you expected to happen"? also, I'm a bit confused - is this section in the bug report supposed to outline what actually happened, what the user expected to happen, or both?

Reply:
It should be both. What happened and how it differs from what was expected. Good point here Jaiveer. That can be clarified

Reply:
how does this sound:  "Tell us what happened and how is it different form the expected?"